lib/ModelMixins/TableMixin.ts

```diff
@@ -270,7 +270,7 @@ function TableMixin<T extends Constructor<Model<TableTraits>>>(Base: T) {
             name: yColumn.title,
             categoryName: this.name,
             key: `key${this.uniqueId}-${this.name}-${yColumn.name}`,
-            type: "line",
+            type: this.chartType,
             xAxis,
```

Comment:
Currently [chartType](https://github.com/TerriaJS/terriajs/blob/line-and-point-custom-chart-component/lib/Traits/DiscretelyTimeVaryingTraits.ts#L37) defaults to `momentLines`. So this change forces the chart type of all existing table items that don't specify one to `momentLines`.

To avoid that, I think we should:
1) Remove the default value for [chartType](https://github.com/TerriaJS/terriajs/blob/line-and-point-custom-chart-component/lib/Traits/DiscretelyTimeVaryingTraits.ts#L37)
2) Change this line to `type: this.chartType || "lineAndPoint"`

I think 1) is alright because the [existing code that uses chartType](https://github.com/TerriaJS/terriajs/blob/line-and-point-custom-chart-component/lib/ModelMixins/DiscretelyTimeVaryingMixin.ts#L335) redundantly sets a default.

Reply:
ah, well caught! that would have been bad.

`chartType` in itself probably needs reconsidering, because a catalog item can theoretically have multiple chart types and why is it in DiscretelyTimeVaryingTraits, etc. But I'll make a ticket for it :)